x-pack/legacy/plugins/siem/cypress/integration/smoke_tests/inspect/inspect.spec.ts

```diff
@@ -4,34 +4,52 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-import { HOSTS_PAGE } from '../../lib/urls';
+import { HOSTS_PAGE, NETWORK_PAGE } from '../../lib/urls';
 import {
-  INSPECT_BUTTON_ICON,
   INSPECT_MODAL,
-  INSPECT_BUTTONS_IN_SIEM,
+  INSPECT_NETWORK_BUTTONS_IN_SIEM,
+  INSPECT_HOSTS_BUTTONS_IN_SIEM,
   TIMELINE_SETTINGS_ICON,
   TIMELINE_INSPECT_BUTTON,
 } from '../../lib/inspect/selectors';
 import { DEFAULT_TIMEOUT, loginAndWaitForPage } from '../../lib/util/helpers';
 import { executeKQL, hostExistsQuery, toggleTimelineVisibility } from '../../lib/timeline/helpers';
+import { closesModal, openHostsAndStatsTables } from '../../lib/inspect/helpers';
 
 describe('Inspect', () => {
-  describe('Hosts and network stats and tables', () => {
-    INSPECT_BUTTONS_IN_SIEM.map(table =>
+  context('Hosts stats and tables', () => {
+    before(() => {
+      loginAndWaitForPage(HOSTS_PAGE);
+    });
+    afterEach(() => {
+      closesModal();
+    });
+
+    INSPECT_HOSTS_BUTTONS_IN_SIEM.map(table =>
```

Comment:
nit: consider using `forEach` instead of `map` when the work being performed in each iteration is for effect (executing a test), as opposed to mapping from `a -> b`

Reply:
Thx!! I was respecting previous implementation 😊